app/src/main/java/com/pysiak/simple/clock/ui/clock/TimezonedbApi.java

```diff
@@ -0,0 +1,12 @@
+package com.pysiak.simple.clock.ui.clock;
+
+import retrofit2.Call;
+import retrofit2.http.GET;
+import retrofit2.http.Query;
+
+public interface TimezonedbApi {
```

Comment:
Move this class to com.pysiak.simple.clock.app.networking package. Also rename it to follow model new class name. See previous comment